pages/docs.html

```diff
@@ -1221,11 +1281,16 @@ <h1 class="main__heading p--1" id="snackbar">
         </main>
     </div>
 
+    <button class="btn--float sidebar-btn">
```

Comment:
You can look into sidebar-btn here and ensure you follow consistent BEM.

Reply:
yeah I was thinking about that. I have named it menu__btn now. does that work?

Reply:
> yeah I was thinking about that. I have named it menu__btn now. does that work?

Looks good!